### DISCUSSION TITLE
1.70 not working on linux

### DISCUSSION THREAD

**Discussion by BSM24** [Syncplay/syncplay#614] [Q&A]
My friend wants to join but is unable to get syncplay working. The config screen comes up and we enter all the stuff thats needed (same as we did as in windows) but the config screen closes and it wont open the room window. The app is also closed when we checked in task manager.

**Comment by Et0h** [comment]
Hi BSM24. Please provide ask your friend to provide us with the following answers:
1. What flavour and version of Linux are they using?
2. What media player are they using? Does the media player run fine when not run with Syncplay? Does Syncplay work fine if a different media player is used?
3. Are they using the AppImage, the Deb Package, or the tarball for Syncplay?
4. If they are running the AppImage or Deb Package, is the problem resolved if they run Syncplay directly using the tarball?
5. If 1.7.0 is broken, then does Syncplay 1.6.9 work fine? How about 1.7.0 Beta?
6. Does it give any error message? Does it give more detail if you run in --debug mode?
7. Does it run fine if Syncplay is run in --no-gui mode?
